src/datasources/asset/components/AssetQueryEditor.tsx

```diff
@@ -66,7 +66,12 @@ export function AssetQueryEditor({ query, onChange, onRunQuery, datasource }: Pr
   return (
     <div style={{ position: 'relative' }}>
       <InlineField label="Query type" labelWidth={22} tooltip={tooltips.queryType}>
-        <Select options={filterOptions} onChange={handleQueryTypeChange} value={queryType} width={85} />
+        <Select
```

Comment:
I guess I missed this in an earlier change, but this "Query type" field doesn't match the UX we have in other data sources. Is that intentional? We've used the `RadioButtonGroup` in the past:
https://github.com/ni/systemlink-grafana-plugins/blob/8a028a82a2688d105eb2e165fcf83a558e2069ba/src/datasources/tag/components/TagQueryEditor.tsx#L38

Reply:
@mure this was already discussed with @TJ-G